packages/sit-onyx/src/components/OnyxMobileNavButton/OnyxMobileNavButton.vue

```diff
@@ -53,38 +66,60 @@ defineSlots<{
 
 .onyx-mobile-nav-button {
   @include layers.component() {
-    display: flex;
-    background-color: var(--onyx-color-base-background-blank);
-    color: var(--onyx-color-text-icons-neutral-medium);
-    padding: var(--onyx-spacing-md);
-    cursor: pointer;
-
-    :where(&) {
-      border: none;
-    }
+    // should be adjusted to the height of the control button
+    --top-position: 3.5rem;
```

Comment:
we also need it to calculate the max heigth of only the flyout, without affecting the backdrop. also the backdrop needs position fixed and would overlap the top-bar without the top-position

Reply:
background info: it already has been implemented like this, see the diff
![image](https://github.com/SchwarzIT/onyx/assets/151019977/798016f2-71fb-4aac-99e6-d52b550d7384)
inside the OnyxNavBar.vue
but I moved all the styles to the mobile nav button now because of separation of concerns/ the nav button owns it's own positioning.